src/vars.c

```diff
@@ -2953,30 +2953,10 @@ static Int InitKernel (
 
     /* install executors, evaluators, and printers for local variables     */
     InstallExecStatFunc( T_ASS_LVAR       , ExecAssLVar);
-    InstallExecStatFunc( T_ASS_LVAR_01    , ExecAssLVar01);
```

Comment:
This is the only place which references `ExecAssLVar01` etc., so this PR should also remove it and its siblings.